packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -250,13 +258,18 @@ class DateTimePicker extends React.Component {
         updatedDateTime.subtract(1, 'hours');
       }
 
-      this.timeValue = DateTimeUtils.formatISODateTime(updatedDateTime.format(), 'HH:mm');
+      if (updatedDateTime) {
+        this.timeValue = DateTimeUtils.formatISODateTime(updatedDateTime.format(), 'HH:mm');
```

Comment:
above we are subtracting 1 hour from the `updatedDateTime` value. Should that logic occur in this check or why do we need that check here? It might help to add an inline comment to describe what this check is validating. I think that it's a non-empty date ?

Reply:
I added comment to indicate that the face-up time input value would need to be updated to account for the missing hour scenario. The check is to make sure we have a valid object before calling the format function on it.

https://github.com/cerner/terra-framework/pull/619/commits/784e52a725189be07a1cc7da27ca567b630385ed